tcms/testcases/static/testcases/js/search.js

```diff
@@ -59,6 +59,10 @@ $(document).ready(function() {
                 params['is_automated'] = false;
             };
 
+            if ($('#id_text').val()) {
+                params['text__icontains'] = $('#id_text').val();
```

Comment:
I think it would be better if it is:
```javascript
const text = $('#id_text').val()
if (text) {
     params['text_icontains'] = text
}
```
this will be faster, because it calls `$` once , hence you won't go to the DOM, search for the element, etc. twice

Reply:
OK, but this might be visually inconsistent with the other part of the code. I can change it if @atodorov wouldn't mind.

Reply:
Toni is right to propose this change. It should be the canonical way since you don't want to evaluate something right after you had evaluated it.  I suspect some tools may start to complain over time about too many variable in this function but we'll deal with that when we have to.